test/public.test.ts

```diff
@@ -35,25 +36,25 @@ describe('Public Inverse REST API Endpoints', () => {
   });
 
   it('getCandles()', async () => {
-    expect(await api.getCandles(instId, '1m')).toMatchObject(
+    expect(await api.getCandles({ instId: instId })).toMatchObject(
```

Comment:
Shouldn't this be getCandlesV2? Same for the fns below

Reply:
Yeah, it should now. I forgot to changed that part when we made new changes.